.github/workflows/build_ffmpeg.yaml

```
@@ -43,3 +43,26 @@ jobs:
         with:
           name: "ffmpeg_${{ matrix.ffmpeg-version }}_linux_x86_64"
           path: artifact_dir
+  LGPL-macOS:
+    strategy:
+      fail-fast: false
+      matrix:
+        ffmpeg-version: ["4.4.4", "5.1.4", "6.1.1", "7.0.1"]
+        runner: ["macos-m1-stable", "macos-12"]
```

Pytorch doesn't support x86 for mac anymore

```suggestion
        runner: ["macos-m1-stable"]
```

---

Oh, interesting. The curious bit here is that my system is an x86 Mac, so if I'm going to do any local testing of Mac wheels, then I'll need to keep this in. I can just use CI for everything instead. Let me know which you think is more important: aligning with general PyTorch, or enabling local testing.

---

You shouldn't need to keep this in for local testing. This just instructs the CI to spin x86 instances. This is never executed on your machine. The actual build script should execute just fine for you locally.

---

Yes, agreed, this never executes on my machine. But I will want to locally define `BUILD_AGAINST_ALL_FFMPEG_FROM_S3` and build on my machine to test that flow locally. And if we don't build non-GPL FFmpeg x86 binaries for Mac in this step in CI, then they won't be available to upload to S3, and I won't be able to test that flow locally.

We can remove it after I've gotten everything setup, but I think at least while I'm working on this, it's better that these binaries are available.